tests/_plugins/ui/_impl/test_table.py

```diff
@@ -307,3 +308,10 @@ def test_table_with_too_many_rows_unknown_total():
     assert table._component_args["has-more"] is True
     assert table._component_args["total-rows"] == "too_many"
     assert len(table._component_args["data"]) == 30
+
+
+def test_table_with_too_many_rows_column_summaries_disabled():
+    data = {"a": list(range(20))}
+    table = ui.table(data, _internal_summary_row_limit=10)
+    summaries = table.get_column_summaries(EmptyArgs())
+    assert summaries.is_disabled is True
```

Comment:
can you add another test that when we search/filter that brings it down to less than 10, summaries is re-enabled?

Reply:
Added. Thanks for reminding!